README.md

```diff
@@ -1,29 +1,32 @@
 # mondolib
-Python library for mondo QC
-
-This has intentionally bespoke and mondo-unique checks
+Python library for Mondo.
```

Comment:
## QC -> general
@cmungall @hrshdhgd Nico and I want to use this repo for some general purpose utils. We aim to put it on PyPi and re-use it in mondo-ingest / elsewhere.